ert_shared/ensemble_evaluator/entity/ensemble_base.py

```diff
@@ -1,6 +1,7 @@
 import asyncio
 import websockets
 
+import ert_shared.ensemble_evaluator.entity.serialization as serialization
```

Comment:
This is probably better written as 
`from ert_shared.ensemble_evaluator.entity import serialization` ? Idk